onnxruntime/python/tools/quantization/execution_providers/qnn/quant_config.py

```diff
@@ -0,0 +1,80 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from pathlib import Path
+
+import onnx
+
+from ...calibrate import CalibrationDataReader, CalibrationMethod
+from ...quant_utils import QuantType
+from ...quantize import StaticQuantConfig
+
+Q16_TYPES = {QuantType.QInt16, QuantType.QUInt16}
+Q8_TYPES = {QuantType.QInt8, QuantType.QUInt8}
+OP_TYPES_TO_EXCLUDE = {"Cast", "Transpose"}
```

Comment:
@HectorSVC This is the list of ops to exclude from `op_types_to_quantize`. Does this look ok to you?

Reply:
I removed Transpose from this list.